app/models/manager_refresh/inventory/automation_manager.rb

```diff
@@ -0,0 +1,66 @@
+module ManagerRefresh::Inventory::AutomationManager
+  extend ActiveSupport::Concern
+  include Core
+
+  class_methods do
+    def has_automation_manager_configuration_scripts(options = {})
+      has_configuration_scripts({
```

Comment:
right so with this approach, I would have has_cloud_manager_vms, has_infra_manager_vms or has_vms and I can pick any of them in Persistor?